src/index.js

```diff
@@ -29,34 +29,42 @@ class Anvil {
     this.limiter = new RateLimiter(this.requestLimit, this.requestLimitMS, true)
   }
 
-  fillPDF (pdfTemplateID, payload) {
-    return this.requestREST(`/api/v1/fill/${pdfTemplateID}.pdf`, {
-      method: 'POST',
-      json: payload,
-      encoding: null,
```

Comment:
All good without specifying an encoding? It doesnt try to translate into UTF8 by default?

Reply:
It is my understanding that for the `request` library, the `encoding: null` option said "I want the response to be a Buffer", which makes sense b/c we wrote the client to deal with buffers as the happy-path response.

With `node-fetch`, you explicitly tell it how you'd like to consume the response via the `response.buffer()` (or similar) calls, and there is no UTF-stuff that happens when dealing with the `stream` or `buffer` types. There is, for example, when you call `response.json()` to consume it.

Anywho, I think we're good and this worked for me no problem locally.

From the `request` documentation:
> `encoding` - encoding to be used on setEncoding of response data. If null, the body is returned as a Buffer. Anything else (including the default value of undefined) will be passed as the encoding parameter to toString() (meaning this is effectively utf8 by default). (Note: if you expect binary data, you should set encoding: null.)

Relevant `node-fetch` docs: https://www.npmjs.com/package/node-fetch#interface-body